packages/kbn-capture-oas-snapshot-cli/src/run_capture_oas_snapshot_cli.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import path from 'node:path';
+import fs from 'node:fs/promises';
+import { encode } from 'node:querystring';
+import fetch from 'node-fetch';
+import { run } from '@kbn/dev-cli-runner';
+import { createTestEsCluster } from '@kbn/test';
+import * as Rx from 'rxjs';
+import { REPO_ROOT } from '@kbn/repo-info';
+import chalk from 'chalk';
+import { kibana } from './kibana';
+
+const OAS_FILE_PATH = path.resolve(REPO_ROOT, './oas_docs/bundle.json');
```

Comment:
Let me know what you think of this location and overall approach, but this was the main idea of this PR.

I expect we'll iterate on this CLI to add more sophisticated capabilities. For example generating OAS for `traditional` or project types (`search`, `security` , `o11y`), this is just to get things started in that direction.